File: nodejs/8 kyu/226.js
// https://www.codewars.com/kata/5302d846be2a9189af0001e4/train/javascript

// Passed

function sayHello(name, city, state) {
  let fullName = name.join(" ");
  let result = `Hello, ${fullName}! Welcome to ${city}, ${state}!`;
  return result;
}

const output = sayHello(["John", "Smith"], "Phoenix", "Arizona");
console.log(output);

File: python/7 kyu/226.py
# https://www.codewars.com/kata/54ff0d1f355cfd20e60001fc/train/python

# Passed

def factorial(n):
    if n < 0 or n > 12:
        raise ValueError()
        
    factorial_result = 1
    for i in range(1, n + 1):
        factorial_result *= i
        
    return factorial_result

output = factorial(3)
print(output)